provisioning/baremetal_secrets.go

```diff
@@ -29,13 +30,16 @@ const (
 	inspectorSecretName      = "metal3-ironic-inspector-password"
 	inspectorUsername        = "inspector-user"
 	tlsSecretName            = "metal3-ironic-tls" // #nosec
-	openshiftConfigNamespace = "openshift-config"
 	openshiftConfigSecretKey = ".dockerconfigjson" // #nosec
-	pullSecretName           = "pull-secret"
 	// NOTE(dtantsur): this is kept here to be able to remove the old
 	// secret when a Provisioning is removed.
 	ironicrpcSecretName = "metal3-ironic-rpc-password" // #nosec
 	baremetalSecretName = "metal3-mariadb-password"    // #nosec
+
+	// OpenshiftConfigNamespace holds the name of the openshift-config namespace.
+	OpenshiftConfigNamespace = "openshift-config"
```

Comment:
nit: let's use a naming scheme consistent with the existing code

Reply:
Thanks for the review!

The existing code used:
```
openshiftConfigNamespace = "openshift-config"
```

and:
```
pullSecretName           = "pull-secret"
```

I needed to make the variables public so that I could use them across packages, and the only way to make a variable in Golang public is by starting them with an upper case letter.

So I'm consistent with the preexisting code. Or what better names do you have in mind for these variables?